src/Search/index.js

```diff
@@ -0,0 +1,125 @@
+/**
+ *  Copyright 2020 Adobe. All rights reserved.
+ *  This file is licensed to you under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License. You may obtain a copy
+ *  of the License at http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing, software distributed under
+ *  the License is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR REPRESENTATIONS
+ *  OF ANY KIND, either express or implied. See the License for the specific language
+ *  governing permissions and limitations under the License.
+ */
+
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+import { useState } from 'react'
+import PropTypes from 'prop-types'
+import { navigate, Link } from 'gatsby'
+import { Index } from 'elasticlunr'
+import { Heading, Text } from '@react-spectrum/text'
+import { Item, Menu } from '../Menu'
+import { Popover } from '../Popover'
+import { SearchField } from '../SearchField'
+
+import './search.css'
+
+const Search = ({ searchIndex = {}, ...props }) => {
+  const [index] = useState(Index.load(searchIndex))
+  const [results, setResults] = useState([])
+  const [items, setItems] = useState([])
```

Comment:
you probably don't need a state hook for the `items` value since the value can be calculated from `results`

Reply:
Good stuff, got rid of it.